src/panels/lovelace/components/hui-marquee.ts

```diff
@@ -30,11 +30,20 @@ class HuiMarquee extends LitElement {
   protected updated(changedProperties: PropertyValues): void {
     super.updated(changedProperties);
 
+    if (changedProperties.has("text") && this._animating) {
+      this._animating = false;
+      this.setAttribute("stopped", "stopped");
```

Comment:
I am not sure if I should just make this a property and reflect it. I went with this as I didn't think it was necessary.

Reply:
Wouldn't it be enough to set `this._animating = false`? Or will it just pause then? I did it like this to prevent repaints but that can be solved in different ways since this will do that too.

Reply:
Move the `animation: marquee 10s linear infinite;` to the `:host([animating]) .marquee-inner` rule

Reply:
Yea just `this._animating = false` just pauses at the current position. Thats why I added the `none` but your suggestions worked